.github/workflows/push.yml

```diff
@@ -12,9 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 name: CI
 
-on: push
+on:
+  push:
+    branches-ignore:
+      - development
```

Comment:
I think we can just drop the trigger on push for all other branches. I don't see much value in it and it just keeps the runners busy.

Reply:
deleted, tnx!